app/src/main/java/com/realityexpander/tasky/agenda_feature/util/ZonedDateTimeToStrings.kt

```diff
@@ -0,0 +1,131 @@
+package com.realityexpander.tasky.agenda_feature.util
```

Comment:
What's the difference between your util package and your layer packages? The functions of this file are all presentation related, so maybe it would make more sense to put these in `agenda_feature->presentation->util` instead 🤔

Reply:
OK, that makes sense. Im still figuring out where to put things.